Sources/Paywalls/Components/PaywallStackComponent.swift

```diff
@@ -20,7 +20,7 @@ public extension PaywallComponent {
 
         public enum Overflow: String, PaywallComponentBase {
             case `default`
-            case scroll
+            case scroll = "scrollable"
```

Comment:
Changing the string representation of the scroll property to make it backwards-incompatible. That way we prevent older versions of the SDK with incorrect behaviour from using it. We'll need to change this in the schema & backend too.